tests/plots/test_mapping.py

```diff
@@ -66,6 +66,34 @@ def test_globe_spheroid():
     assert globe_params['a'] == 6367000
     assert globe_params['b'] == 6360000
 
+    
+def test_aea():
+    """Test handling albers equal area projection."""
+    attrs = {'grid_mapping_name': 'albers_conical_equal_area', 'earth_radius': 6367000,
+             'standard_parallel': [20, 50]}
+    proj = CFProjection(attrs)
+
+    crs = proj.to_cartopy()
+    assert isinstance(crs, ccrs.AlbersEqualArea)
+    assert crs.proj4_params['lat_1'] == 20
+    assert crs.proj4_params['lat_2'] == 50
+    assert crs.globe.to_proj4_params()['ellps'] == 'sphere'
+
+
+def test_aea_minimal():
+    """Test handling albers equal area projection with minimal attributes."""
+    attrs = {'grid_mapping_name': 'albers_conical_equal_area'}
+    crs = CFProjection(attrs).to_cartopy()
+    assert isinstance(crs, ccrs.AlbersEqualArea)
+
+
+def test_aea_single_std_parallel():
+    """Test albers equal area with one standard parallel."""
+    attrs = {'grid_mapping_name': 'albers_conical_equal_area', 'standard_parallel': 25}
+    crs = CFProjection(attrs).to_cartopy()
+    assert isinstance(crs, ccrs.AlbersEqualArea)
+    assert crs.proj4_params['lat_1'] == 20
```

Comment:
This does not seem to match with the attrs that was set (attrs says 25, this says 20), which is causing test suite to now fail after the fixes I'm working on in https://github.com/Unidata/MetPy/pull/1325. @dopplershift Can I update it there to get tests passing?

Reply:
*sigh* Yes. This is why we need CI working, because I suck at code review.